gp_learner.py

```diff
@@ -837,8 +955,14 @@ def mutate(
     else:
         children = [child]
 
-
-    # TODO: deep & narrow paths mutation
+    helper = []
+    for child in children:
```

Comment:
i'd actually only do this if not pb_fv, as otherwise we might have n*m children here...